doc/Type/DateTime.pod6

```diff
@@ -252,6 +252,51 @@ Returns an L<Instant|/type/Instant> object based on the invocant.
 
     say DateTime.new('2015-12-24T12:23:00+0200').Instant; # OUTPUT: «Instant:1450952616␤»
 
+=head2 method day-fraction
+
+Defined as:
+
+    method day-fraction(DateTime:D: --> Real:D)
+
+Returns the instant's time as a fraction of a 24-hour day.
+
+    say DateTime.new('2021-12-24T12:23:00.43Z').day-fraction; # OUTPUT: «0.5159772␤»
+
+Notice the C<day-fraction> value is the same as the fractional part of
+the C<modified-julian-date> for the same instant.
+
+=head2 method julian-date
+
+Defined as:
+
+    method julian-date(DateTime:D: --> Real:D)
+
+Returns the L<Julian Date|https://en.wikipedia.org/wiki/Julian_day> (JD) for the UTC date and time.
```

Comment:
Are you sure about capitalizing Date?

Reply:
I was actually referring to the DateTime object as an instant, which I believe it is.